static/index.html

```diff
@@ -14,6 +14,27 @@
         <script src="http://code.jquery.com/jquery-1.11.3.min.js"></script>
         <script src="http://knockoutjs.com/downloads/knockout-3.3.0.js"></script>
 
+        <style>
```

Comment:
Yeah I think it's small enough to have it embedded and we want to keep it small.